db/migrate/20250202194446_add_myturn_meta_to_reservable_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddMyturnMetaToReservableItems < ActiveRecord::Migration[7.2]
+  def change
+    add_column :reservable_items, :myturn_metadata, :jsonb, default: "{}"
```

Comment:
Might want to double check, but does the default of `"{}"` actually wind up as literally a string?

Reply:
You're right, it does end up as a string. Using a plain empty Hash works as intended, though. Good catch!